lib/round.rb

```diff
@@ -0,0 +1,70 @@
+class Round
+
+  attr_reader :deck, :card, :turn, :turns, :turn_index, :guesses_correct
```

Comment:
you should only have attr_readers for the variables that you want to call outside of your class. The only variables you want to call outside of your class are deck and turns... so you can get rid of the rest of these since you can refer to the instance variable within the class to get what you need.